src/btree/iter.rs

```diff
@@ -53,8 +57,8 @@ struct PendingBackend {
 pub enum LastKey {
 	Start,
 	End,
-	At(Vec<u8>),
-	Seeked(Vec<u8>),
+	At(ValuePtr),
+	Seeked(ValuePtr),
```

Comment:
I am a bit hesitant to switch this one to rc as it changes the iterator api. Seeked is not really useful, At will be to save one key memory per iterator and a mem copy of the key on all alloc (could use an allocated buffer but not sure if relevant) but the iterator is nowhere near performant or trying to be.
Personally I would rather keep the rc to the change overlay only.

Reply:
Could you elaborate on which API to keep unchanged? I just tried reverting it here but it does not change any API signature

Reply:
Just iterator `next` and `prev` returns IterResult that used to return plain Vec (as used in the fuzzer, I mean the change to the fuzzer could also be reverted then).

Reply:
@cheme Done. Please take another look